src/node.cc

```diff
@@ -4114,10 +4114,11 @@ inline void PlatformInit() {
       continue;
     // Anything but EBADF means something is seriously wrong.  We don't
     // have to special-case EINTR, fstat() is not interruptible.
-    // if (errno != EBADF)
-    //   ABORT();
-    if (fd != open("/dev/null", O_RDWR))
+    if (errno != EBADF)
       ABORT();
+    open("/dev/null", O_RDWR);
```

Comment:
I guess this related to "AIX: parallel/test-stdio-closed #8375".

On AIX, I remember that there is a kernel parameter regarding max number of pseudo terminals allowed in the system. I suspect this issue related to the kernel parameter. Need to check the "errno" when the test fails.